robo.yml

```diff
@@ -24,7 +24,7 @@ data:
 
   - name: "digital-europa-thesaurus"
     graph: "http://data.europa.eu/uxp"
-    url: "http://publications.europa.eu/resource/cellar/8028fdf8-8d0f-11ee-8aa6-01aa75ed71a1.0001.03/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/fdeb7b20-5c6e-11ee-9220-01aa75ed71a1.0001.03/DOC_1"
```

Comment:
This change was done for reverting update in EWPP-3798 as it has issue in data from OP side.